.travis.yml

```diff
@@ -57,3 +57,4 @@ notifications:
   email:
     on_success: never
     on_failure: always
+  webhooks: https://w3c-test.org/prbuildbot.py
```

Comment:
So. I'm not really thrilled by this being in a separate repo. I was thinking of moving the check_stability.py script to `tools/ci` (the `tools` submodule will be inlined as soon as the CSS tests are inlined), so I think I'd like this to live under `tools/ci/` too. (I think the name `prbuildbot` is a little confusing because `buildbot` is a CI system that's unrelated to this).